ietf-wson-topology.yang

```diff
@@ -233,318 +156,11 @@ module ietf-wson-topology {
     uses wson-node-attributes;
   }
 
-  augment "/nw:networks/nw:network/nw:node/tet:te"
-        + "/tet:tunnel-termination-point" {
-    when "/nw:networks/nw:network/nw:network-types"
-       +"/tet:te-topology/wson:wson-topology" {
-      description "This augment is only valid for WSON";
-    }
-    description "WSON tunnel termination point augmentation";
-    uses wson-ttp-attributes;
-  }
-
   /*
-   * Augment TE bandwidth
+   * Augment TE bandwidth is removed to be solved in next version. 
    */

```

Comment:
I am not sure whether we need or not to add these augmentations in the next version. Better to remove this comment.